txmongo/connection.py

```diff
@@ -16,6 +16,10 @@
 from txmongo.protocol import MongoProtocol, Query
 
 
+DEFAULT_MAX_BSON_SIZE = 16 * 1024**2
```

Comment:
Do we really want to do the calculation here? Why can't we just use: 

DEFAULT_MAX_BSON_SIZE = 16777216  # 16MiB